content/en/tracing/guide/force_tracing.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Force Tracing
+kind: documentation
+aliases:
+  - /tracing/guide/force_tracing
+---
+
+
+APM enables priority sampling by default to allow traces between two Datadog endpoints to be sampled together. This prevents trace sampling from removing segments of a distributed trace (i.e. ensures completeness) and helps removing unimportant ones. You can override this functionality to force keep a trace (critical transaction, debug mode, etc.) or force drop a trace (health checks, static assets, etc) by the agent and the server using forced tracing.
+
+Forced tracing should be done only before any context propagation. If this happens after the propagation of a context, the system can’t ensure that the entire trace is sampled properly.
+
+
+{{< tabs >}}
+{{% tab "Java" %}}
+
+Manually keep or drop a trace:
+
+```java
+import datadog.trace.api.DDTags;
+import datadog.trace.api.interceptor.MutableSpan;
+import datadog.trace.api.Trace;
+import io.opentracing.util.GlobalTracer;
+
+public class MyClass {
+    @Trace
+    public static void myMethod() {
+        // grab the active span out of the traced method
+        MutableSpan ddspan = (MutableSpan) GlobalTracer.get().activeSpan();
+
+        // ask the sampler to keep the current trace
+        ddspan.setTag(DDTags.MANUAL_KEEP, true);
+
+        // ask the sampler to drop the current trace
+        ddspan.setTag(DDTags.MANUAL_DROP, true);
+
+        // method impl follows
+    }
+}
+```
+
+[1]: /tracing/languages/java/#configuration
+{{% /tab %}}
+{{% tab "Python" %}}
+
+Manually keep or drop a trace:
+
+```python
+from ddtrace import tracer
+from ddtrace.constants import MANUAL_DROP_KEY, MANUAL_KEEP_KEY
+
+@tracer.wrap()
+def handler():
+    span = tracer.current_span()
+
+    # Manually choose to drop healthcheck traces
+    if is_healthcheck():
```

Comment:
Do we need these checks - is_healthcheck()? If it is only as a sample and not a method DD provides - I will suggest removing it as healthcheck is one example, there can be other use cases as well.

Reply:
sure, we don't need `is_healthcheck()` here, I was just trying to give a realistic example of when you would manually choose to drop a trace.

Same with `is_error()`.

The goal was to not encourage people to manually keep or drop everything.